fido2-net-lib/Fido2NetLib.csproj

```diff
@@ -3,7 +3,8 @@
   <Import Project="../lib/Chaos.NaCL/Chaos.NaCL.Include.props" />
 
   <PropertyGroup>
-    <TargetFrameworks>netstandard2.0;net472;netcoreapp2.0</TargetFrameworks>
+    <TargetFrameworks>netstandard2.0;netcoreapp2.0</TargetFrameworks>
+    <TargetFrameworks Condition="'$(OS)' != 'Unix'">netstandard2.0;net472;netcoreapp2.0</TargetFrameworks>
```

Comment:
@idaviddesmet This change would be nice to remove from this PR

Reply:
Whoops! That must have slipped through...
I will remove it from this PR and create another one for it. This line is needed so it builds in macOS and Linux otherwise the compiler will argue about the missing .NET Framework 4.7.2 SDK in those platforms.